common/installer/internal/algo/ubuntu20_4k8s.go

```diff
@@ -123,7 +124,9 @@ BUNDLE_PATH=$BUNDLE_DOWNLOAD_PATH/$BUNDLE_ADDR
 swapon -a && sed -ri '/\sswap\s/s/^#?//' /etc/fstab
 
 ## enable firewall
-ufw enable
+if command -v ufw >>/dev/null; then
```

Comment:
Is this is to check if the `ufw` executable exists?

Reply:
Yes, so that it works on docker host as well (as docker host don’t have ufw command)